### PR TITLE
Desktop-portfolio

### PR DESCRIPTION
**Pull request by Hazard10-eng**
Adding HTML for a desktop portfolio
Adding CSS for styling the desktop portfolio
Running Linters for desktop-portfolio

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -433,3 +435,161 @@ li {
   border-radius: 8px;
   color: #fff;
 }
+
+@media only screen and (min-width: 600px) {
+  .menu {
+    display: none;
+  }
+
+  .navigation {
```
Comment (roshan-bajgain) [inline styles.css]: - [ ] You should use the media query with a min-width property that should range from 768px or wider). Please use this 
`@media screen and (min-width: 768px) { }`

File: index.html [hunk 2]
```diff
@@ -28,6 +28,15 @@
...
+      <nav>
+       <ul class="navigation">
+        <li class="item-navigation">
+          <a href="#">Portfolio</a>
+          <a href="#">About</a>
+          <a href="#">Contact</a>
+        </li>
+       </ul>
+      </nav>
     </header>
```
Comment (roshan-bajgain) [inline index.html:40]: - [ ] According to the project requirement you should stick to the design as much as possible. Please adjust the spacing of the logo to the right as shown in the figure also there should be a gap between the header and section of the content. Please see the design below for reference.
- [ ] Also the background image is missing please download the background image and use it.
### Your design
his


### Figma design
fimga

File: index.html [hunk 3]
```diff
@@ -54,12 +63,12 @@ <h2 class="name">
...
-          src="./image/Snapshoot Portfolio.svg"
-          alt="tonic"
-          class="project-img"
-        />
+       <div class="project tonic-project">
+          <img
+            src="image\Snapshoot Portfolio11.png"
+            alt="tonic"
+            class="project-img"
+          />
```
Comment (roshan-bajgain) [inline index.html:71]: - [ ] You should stick to the design as much as possible. As I notice your image is not showing in the project please fix this issue and also please adjust the gap between the heading of the cards and the list. Please see the image below for reference.
- [ ] Also the color used for the list is different please fix so.
- [ ] I would suggest please implement the same changes for all the cards as there is the same issue for all the cards.
### Your design
his


### Figma design
fimga

File: index.html [hunk 4]
```diff
@@ -109,78 +113,83 @@ <h4 class="project-client">Canopy</h4>
...
+        <h2 class="name">About<br/>Myself</h2>
         <p class="text">
           Hello I’m a software developer! I can help you build a product ,
           feature or website Look through some of my work and experience! If you
           like what you see and have a project you need coded, don’t hestiate to
           contact me.
         </p>
 
-        <div class="contact">
+
```
Comment (roshan-bajgain) [inline index.html]: - [ ] You should stick to the design as much as possible. please fix the spacing as shown in the figure below and also use the same color for `hx` tag also fix the spacing from left and right.
- [ ] Also the image for javascript is missing please fix it.
### Your design
his

### Figma design
fimga

File: index.html [hunk 5]
```diff
@@ -278,7 +287,7 @@ <h4 class="contact-title">Contact me</h4>
         ></textarea>
         <button class="fbutt">Get It Touch</button>
       </form>
+    </div>
```
Comment (roshan-bajgain) [inline index.html:293]: - [ ] You should stick to the design as much as possible. Please fix the spacing and also the elements of the form are centered on the page including the Get in touch button.
- [ ] The background is missing please fix this issue also.
### Your design
his

### Figma design
fimga

File: index.html [hunk 6]
```diff
@@ -109,78 +113,83 @@ <h4 class="project-client">Canopy</h4>
...
-            <span class="pro-lang">Back End Dev</span>
+            <span class="pro-lang">Lead Developer</span>
             <img src="./image/Counter.svg" alt="counter" />
-            <span class="year">2015</span>
+            <span class="year">2018</span>
           </nav>
           <p class="project-description">
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
+            A smart assistant to make driving more safe, efficient, and fun by
```
Comment (roshan-bajgain) [inline index.html]: - [ ] [Optional] I would suggest you please use the active, hover, and pressed for your button design.
- [ ] [Optional] Also please remove the background used for buttons here and make sure that when user hover and pressed then only is changes.
### Your design
his

### Figma design for buttons.

fimga

File: styles.css (hunk 1 above)
Comment (roshan-bajgain) [inline styles.css]: - [ ] In the mobile version the nav list should be hidden It should be shown in only the desktop version. Please fix it
### Your design
his

File: styles.css [hunk 7]
```diff
@@ -433,3 +435,161 @@ li {
...
+    display: flex;
+    flex-direction: row;
+    margin-bottom: 10%;
+    left: 0;
+    right: 0;
+    width: 1056px;
+    height: 456px;
+    margin-right: 142px;
+    margin-left: 142px;
+  }
```
Comment (roshan-bajgain) [inline styles.css]: - [ ] In the mobile version the image should be shown first then there should be cards also fix this issue. Also fix the color of the first list in all the cards
### Your design
your

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @Hazard10-eng, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Required Changes ♻️
- [ ] If you fix all the desktop version issues then it will be ok for the mobile version also. 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 8]
```diff
@@ -192,7 +204,7 @@ <h2 class="name">About<br />Myself</h2>
             <li><img src="./image/Icon - Medium.png" alt="Mail" /></li>
           </ul>
           <button class="resume">Get my resume</button>
-        </div>
+       </div>
         <div class="languages">
```
Comment (iambenkis) [inline index.html:208]: - [ ] Kindly make sure these icons stick to the Figma design, it means their padding should be as in the Figma design. Adjust the padding top.

Screenshot from 2023-01-20 22-40-02

File: index.html [hunk 9]
```diff
@@ -108,79 +118,81 @@ <h4 class="project-client">Canopy</h4>
...
           </p>
           <ul class="pro-tool">
             <li class="tool">html</li>
+            <li class="tool">Ruby on rails</li>
             <li class="tool">css</li>
             <li class="tool">javascript</li>
           </ul>
-          <button class="more">see project</button>
+          <button class=" more more-about">see project</button>
         </div>
```
Comment (iambenkis) [inline index.html:150]: - [ ]  [**OPTIONAL**] Good job adding buttons properly in your project as requested :clap: , I will suggest you add the button style (hover) to all the buttons :+1: 

Screenshot from 2023-01-20 22-37-12

File: styles.css [hunk 10]
```diff
@@ -108,9 +113,10 @@ li {
 
 .work {
   background: #e5e5e5;
-  display: grid;
+  display: flex;
+  flex-direction: column;
   grid-template-columns: 1fr;
-  padding-top: 20%;
+  padding-top: 10%;
```
Comment (iambenkis) [inline styles.css:119]: - [ ] [**OPTIONAL**] To center this block will make your web page more beautiful, and you will stick to the design , kindly make sure it centered :call_me_hand: 

Screenshot from 2023-01-20 22-39-04

File: index.html [hunk 11]
```diff
@@ -234,7 +246,7 @@ <h4 class="tab-title">Skills</h4>
               </a>
             </li>
           </ul>
-        </div>
+      </div>
```
Comment (iambenkis) [inline index.html:249]: - [ ] [**OPTIONAL**]  Kindly Remove this overflow to stick to the design :+1: 

Screenshot from 2023-01-20 22-40-36

**Review by iambenkis — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @Hazard10-eng  
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @iambenkis  when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by GedeonTS — APPROVED** [review]
# STATUS: APPROVED :tada: :clap: :champagne: 
Hi @Hazard10-eng ,

You are amazing :smiley: :clap: !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

 

## Highlights :clap: :tophat: 
- No linter errors :+1: 
- Good README file :+1: 
- Good design :+1: 
## Optional suggestions :small_airplane: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**